build.R

```diff
@@ -1,14 +1,15 @@
 local({
   if (!file.exists('_config.yml')) return()
   x = iconv(readLines('_config.yml', encoding = 'UTF-8'), 'UTF-8')
+  baseurl = grep('^baseurl:', x, value = TRUE)
+  baseurl = sub(".*['\"](.*)['\"].*", "\\1",  baseurl)
```

Comment:
Since the Prof is not here, how about the "evil" `:::`? Does `baseurl = servr:::jekyll_config('.', 'baseurl', '/')` work?